app/templates/components/twilio-make-call.hbs

```diff
@@ -0,0 +1,11 @@
+<div id="twilio_token" data-token=""></div>
+
+<span id="phoneNumber" data-mobile={{mobileWithPrefix}} class="tel_link {{if hidden 'hidden'}}" {{action "makeCall" on='click'}}>
+  <span class="phone_link"><i class="fa fa-phone"></i> {{displayNumber}}</span>
+</span>
+
+{{!-- HANG-UP BUTTON --}}
+<span id="hangup" class="fa-stack fa hangup-call" {{action "hangupCall" on='click'}}>
```

Comment:
@swatijadhav Be good to show/hide this using a component variable ```callActive``` (or something) instead of directly manipulating via jQuery